project_stage_closed/models/task_stage.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields, api
```

Comment:
@dreispt I follow this recommendation https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#imports
I had another remark from @StefanRijnhart as well on the topic. 
If I misinterpreted the guideline or if there is a better way to enforce, please let me know (less work for me ;))